### ISSUE TITLE
Clarify licensing of material

### ISSUE THREAD

**Issue by haesleinhuepf** [ome/www.openmicroscopy.org#514]
Dear OME folks,

first of all, thanks for having all these tons of omero teaching materials online. You're doing fantastic work. It's just, may I ask you to clarify licensing of your guides and tutorials? 

For example, when I copy code, images and figures from one of your repositories which have no license
* https://github.com/ome/omero-guide-ilastik
* https://github.com/ome/minimal-omero-client
* https://github.com/ome/omero-guide-fiji
* https://github.com/ome/omero-guide-*
* https://github.com/ome/guide-template
* https://github.com/ome/training-scripts
* https://github.com/ome/omero-guide-upload
* https://github.com/ome/omero-deployment-examples
* https://github.com/ome/omero-matlab
* https://github.com/ome/infrastructure
* https://github.com/ome/omero-web-docker

I feel a bit bad because no license typically means, copying is not allowed. If the material would be licensed, e.g. with CC-BY, I would know how to properly cite / acknowledge your work and it would be compatible with my stuff, which I typically license CC-BY. Furthermore, example code shared under GPL cannot be put into CC-BY resources. These two are incompatible. I can also not put example code license GPL in my code which is typicall BSD. There is also some material where the github repository is GPL licensed or has no license and the website rendered from it is CC-BY licensed. Maybe that could be clarifed as well:
* https://github.com/ome/bio-formats-documentation
* https://github.com/ome/omero-documentation
* https://github.com/ome/bio-formats-examples

Last but not least, this license file is broken:
https://github.com/ome/omero-iviewer/blob/master/LICENSE

Thanks again for maintaining these fantastic resources! :-)

Best,
Robert

**Comment by sbesson** [comment]
@haesleinhuepf thanks for your feedback, your support and raising these important issues to our attention.

Answering shortly, the spirit of both the documentation sets (technical and walk-throughs), the deployment tools as well as the code examples is to be as permissive as possible i.e. BSD-2 or CC-BY depending on the use case.

For most of the repositories, I think the issue is:
- either that the license is  missing or broken
- or that the license is internally inconsistent e.g. bio-formats-examples is BSD-2 as per the POM

Most of these are oversights for historical reasons. We will start reviewing our repositories using the list you mentioned above as a starting point and either making the change directly or open issues if more discussion is required. Is there any particular repository you are currently blocked on and would need prioritizing?

**Comment by haesleinhuepf** [comment]
> Is there any particular repository you are currently blocked on and would need prioritizing?

No, not urgently. I worked around it. I just recently hit the issue when I wanted to write a CC-BY blog post cooying some code from here:
https://github.com/ome/omero-guide-ilastik

But again, no hurry. 🌞

Have a great weekend!

Robert
